index.html

```diff
@@ -28,61 +28,6 @@ <h1 class="main-title">Light Your Heart on Fire</h1>
 
   </section>
 
-
-  <!-- Begin Mailchimp Signup Form -->
-
-  <section class="hero">
-    <div class="newsletter-signup">
-
-      <div id="mc_embed_signup">
-      <form action="https://gmail.us5.list-manage.com/subscribe/post?u=5b441666cd710919cbcaa86e8&amp;id=d37fc91c90" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
-          <div id="mc_embed_signup_scroll">
-      	<label for="mce-EMAIL">Join the newsletter</label> <br>
-      	<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="Enter your email address‍" required>
-          <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
-          <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_5b441666cd710919cbcaa86e8_d37fc91c90" tabindex="-1" value=""></div>
-          <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
-          </div>
-      </form>
-      </div>
-
-      <!--End mc_embed_signup-->
-
-<!-- UNADULTERED MAILCHIMP CODE SNIPPET -->
-
-<!-- Keli's questions:
-1. I don't think I need line 59 for functinality so I got rid of it completely and moved all styling to my css folder- is this line purely style related?
-2. What is line 72 referring to and can you explain it to me?
-3. How can I get the email form and button on the same line?  -->
```

Comment:
I would probably do that using flex box.

adding this to your stylesheet (or modifiying your stylesheet if you already have styles for `#mc_embed_signup_scroll`)

```
#mc_embed_signup_scroll {
  display: flex;
  align-items: center;
}
```

should do it.